fe/fe-core/src/main/java/org/apache/doris/nereids/util/JoinUtils.java

```diff
@@ -404,4 +404,15 @@ public static boolean checkReorderPrecondition(LogicalJoin<?, ?> top, LogicalJoi
         markSlots.retainAll(bottom.getOutputSet());
         return markSlots.isEmpty();
     }
+
+    public static boolean isSemiOrAntiJoin(JoinType joinType) {
```

Comment:
duplicates to JoinType.isSemiOrAntiJoin()

Reply:
done